force-app/main/default/classes/StaticTask.cls

```diff
@@ -18,85 +16,90 @@ public with sharing class StaticTask {
     return emailToContactMap;
   }
 
-  public static void setContactsToMap(List<String> contacts){
+  public static void setContactsToMap(List<String> contacts) {
     for (String contactAsString : contacts) {
       Contact parsedContact = new Contact();
-      List<String> contactSeparatedByComaString = getContactSeparatedByComaString(contactAsString);
+      List<String> contactSeparatedByComaString = getContactSeparatedByComaString(
+        contactAsString
+      );
       setContactToMap(contactSeparatedByComaString, parsedContact);
     }
   }
 
-  public static void setContactToMap(List<String> contactSeparatedByComaString, Contact parsedContact){
+  public static void setContactToMap(
+    List<String> contactSeparatedByComaString,
+    Contact parsedContact
+  ) {
     for (String element : contactSeparatedByComaString) {
       setFirstNameField(element, parsedContact);
       setLastNameField(element, parsedContact);
       setEmailField(element, parsedContact);
 
-      if(isEmailNotEmpty(element)){
+      if (isEmailNotEmpty(element)) {
         addContactToMap(parsedContact, emailToContactMap);
       }
     }
   }
 
-  public static void setEmailField(String elem, Contact contact){
+  public static void setEmailField(String elem, Contact contact) {
```

Comment:
the same question as below about alike methods.

_Hint_: think of other collections